nablapps/core/views.py

```diff
@@ -147,15 +147,20 @@ def _add_forum(self, context):
                         break
             context["new_forum_messages"] = new_forum_messages
 
-            try:
-                fysmat_class = FysmatClass.objects.get(user=self.request.user)
-                class_channel = Channel.objects.get(group=fysmat_class, is_class=True)
+            if FysmatClass.objects.filter(user=self.request.user).exists():
+                fysmat_class = FysmatClass.objects.filter(user=self.request.user)[
+                    0
```

Comment:
Change do use `.first`

Reply:
Should be changed now, in the most recent commit :)